lib/sequent/test/command_handler_helpers.rb

```diff
@@ -116,33 +116,31 @@ def deserialize_events(events)
             Class.const_get(type).deserialize_from_json(Sequent::Core::Oj.strict_load(json))
           end
         end
-
       end
 
       def given_events *events
-        @event_store.given_events(events)
+        @event_store.given_events(events.flatten(1))
```

Comment:
It's because we want to do `given_events(events_list1 + events_list2)` with no more parentheses and splats :smiley_cat: